tests/unit/core/test_addonManager.js

```diff
@@ -474,6 +492,7 @@ describe(__filename, () => {
         guid,
         needsRestart,
         status: addonManager.getAddonStatus({ addon: fakeAddon }),
+        canUninstall: true,
```

Comment:
How do we know this value will be `true`? Is it based on a default, and if so, should we be setting something explicitly in the test, rather than relying on a default?

Reply:
It is the default value of `fakeClientAddon()` in this case. I'll update the test case to reflect that.